tensorflow_datasets/core/download/kaggle.py

```diff
@@ -135,7 +139,7 @@ def download_file(self, fname, output_dir):
     return os.path.join(output_dir, fname)
 
 
-def _run_kaggle_command(command_args, competition_name):
+def _run_kaggle_command(command_args: List[str], competition_name: str) -> None:
```

Comment:
`command_args` can be dict or tuple or list. So, It is safe to put it `Any`. 

Also, in try block, `tf.compat.as_text(output)` is returned , So, output annotation will not be `None`, It should be `Optional[str]`. Refer doc [here](https://www.tensorflow.org/api_docs/python/tf/compat/as_text?version=nightly)